.github/workflows/publish.yml

```
@@ -32,7 +33,10 @@ jobs:
           if ! git diff --exit-code docs/antora.yml; then
             git config user.name "${GITHUB_ACTOR}"
             git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+            git checkout -b switch-docs-version
             git add docs/antora.yml
             git commit -m "Switch docs version back"
-            git push
+            git push -u origin switch-docs-version
+            gh pr create --title "Switch docs version back" --body "" \
```

There’s a shortcut to fill the PR details from the commit message:

```suggestion
            gh pr create --fill \
```

---

Should we add assignee, so it’s less likely we’ll overlook the PR?
```sh
              --assignee @rubocop/rubocop-rspec \
```

---

I think CODEOWNERS will automatically add it to the assigny. Am I missing something?

---

No, I think you are right. I sometimes overlook _issues_, but I think get notifications for all PRs due to CODEOWNERS. Thanks for pointing it out.